packages/pigeon/pigeons/core_tests.dart

```diff
@@ -87,6 +143,43 @@ abstract class HostIntegrationCoreApi {
   @ObjCSelector('echoUint8List:')
   Uint8List echoUint8List(Uint8List aUint8List);
 
+  // ========== Syncronous nullable method tests ==========
+
+  /// Returns the inner `aString` value from the wrapped object, to test
+  /// sending of nested objects.
+  @ObjCSelector('extractNestedNullableStringFrom:')
+  String? extractNestedNullableString(AllNullableTypesWrapper wrapper);
+
+  /// Returns the inner `aString` value from the wrapped object, to test
+  /// sending of nested objects.
+  @ObjCSelector('createNestedObjectWithNullableString:')
+  AllNullableTypesWrapper createNestedNullableString(String? nullableString);
```

Comment:
Similarly I would consider leaving this using non-nullable versions unless you specifically plan to do null testing with this one.